docs/how-to/dss/setup-nvidia-drivers.rst

```diff
@@ -0,0 +1,34 @@
+.. _setup-nvidia-drivers:
+
+Setup NVIDIA Drivers
```

Comment:
@mvlassis There is already an ongoing PR adding this section. See [here](https://github.com/canonical/data-sciencestack/pull/131). I already gave my feedback there - so let's remove this page from this PR.

@kimwnasptd this is the kind of things that can happen and we have to avoid when two people work on the same docs task, as we were discussing in our previous meeting. That's why I advocate for only one person per task, especially when dealing with changes in the same page.

Reply:
The file has been removed, as it addressed by PR #131.